pom.xml

```diff
@@ -55,11 +55,49 @@
 
   <dependencies>
     <dependency>
+     <groupId>org.jenkins-ci.main</groupId>
+     <artifactId>jenkins-core</artifactId>
+     <version>2.63</version>
+     <scope>provided</scope>
+    </dependency>
+    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>dashboard-view</artifactId>
       <version>2.0</version>
       <optional>true</optional>
     </dependency>
+    <!-- dependencies on Jenkins Pipeline plugins -->  
+    <dependency>  
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>  
+      <artifactId>workflow-job</artifactId>  
+      <version>1.14</version>  
```

Comment:
I will check.
In fact most of dependecies was a result of experiments.
I will try to play more with dependecies.
However if plugin adds functionality to pipeline type of job shouldn't it have dependency to pipeline?